addOns/retire/retire.gradle.kts

```diff
@@ -0,0 +1,24 @@
+version = "0.1.0"
+description = "Retire.js"
+
+zapAddOn {
+    addOnName.set("Retire.js")
+    zapVersion.set("2.9.0")
+
+    manifest {
+        author.set("Nikita Mundhada and the ZAP Dev Team")
+        bundle {
+            baseName.set("org.zaproxy.addon.retire.resources.Messages")
+            prefix.set("retire")
+        }
+        bundle {
+            baseName.set("org.zaproxy.addon.retire.resources.help%LC%.helpset")
```

Comment:
23200 [AWT-EventQueue-0] ERROR org.zaproxy.zap.control.AddOnInstaller  - Declared bundle not found in retire add-on:
java.util.MissingResourceException: Can't find bundle for base name org.zaproxy.addon.retire.resources.help%LC%.helpset, locale en_GB
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2055)

Reply:
Needs also:
```kts
localeToken.set("%LC%")
```

Reply:
Add be under `helpSet`.

Reply:
I had, had the localeToken.set but in the a bundle section and it wouldn't import. Now I "think" I'm clear on how it works. Thanks!